win/win-tts/Program.cs

```diff
@@ -12,12 +15,44 @@ static void Main(string[] args)
             var synth = new SpeechSynthesizer();
             var stream = new MemoryStream();
             var format = new SpeechAudioFormatInfo(16000, AudioBitsPerSample.Sixteen, AudioChannel.Mono);
+            SelectVoice(synth);
             synth.SetOutputToAudioStream(stream, format);
             synth.SpeakSsml(args[0]);
 
             var stdout = Console.OpenStandardOutput();
             stream.Position = 0;
             stream.CopyTo(stdout);
         }
+
+        private static void SelectVoice(SpeechSynthesizer synth)
+        {
+            StringBuilder sb = new StringBuilder();
+            string voice = ConfigurationManager.AppSettings.Get("voice");
+            if (voice != null)
+            {
+                try
+                {
+                    synth.SelectVoice(voice);
+                }
+                catch (ArgumentException ex)
+                {
+                    sb.AppendLine(ex.Message);
+                    sb.AppendLine("Using the default voice.");
+                }
+            }
+            AddInfoForAvailableVoices(sb, synth);
+            File.WriteAllText("win-tts.log", sb.ToString());
```

Comment:
We could write the error to stderr instead, since everything that we write to stderr will be logged into the already existing DATIS.log

Reply:
I did write to stderr first, but as I was starting the the application standalone, I could not see the output on the console. I can change this to stderrr again. Wasn't aware that this is captured by DATIS.